CONTRIBUTING.quickstart.md

```diff
@@ -0,0 +1,294 @@
+# Cypress Contributor Quick Start Guide
+
+Welcome! This guide gets you coding on Cypress in under 15 minutes. Skip the overwhelming details and focus on the essentials to start contributing effectively.
+
+## 🚀 Quick Setup (5 minutes)
+
+### Prerequisites
+```bash
+# Check your Node.js version (required: ^20.1.0 || ^22.0.0 || >=24.0.0)
+node --version
+
+# If wrong version, use nvm
+nvm install 22.0.0
+nvm use 22.0.0
```

Comment:
Attempting to run `yarn install` with Node.js `22.0.0` (assuming that Yarn v1 Classic is already installed) results in the error message:

```text
$ yarn install
yarn install v1.22.22
[1/6] Validating package.json...
error cypress@15.6.0: The engine "node" is incompatible with this module. Expected version ">=22.19.0". Got "22.0.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```